Userland/Libraries/LibGfx/ImageFormats/TIFFMetadata.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023, Lucas Chollet <lucas.chollet@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Variant.h>
+#include <AK/Vector.h>
+#include <LibGfx/Size.h>
+
+namespace Gfx {
+
+namespace TIFF {
+
+enum class Type {
+    Byte = 1,
+    ASCII = 2,
+    UnsignedShort = 3,
+    UnsignedLong = 4,
+    UnsignedRational = 5,
+    Undefined = 7,
+    SignedLong = 9,
+    SignedRational = 10,
+    Float = 11,
+    Double = 12,
+    UTF8 = 129,
+};
+
+template<OneOf<u32, i32> x32>
+struct Rational {
+    using Type = x32;
+    x32 numerator;
+    x32 denominator;
+};
+
+using Value = Variant<u8, String, u16, u32, Rational<u32>, i32, Rational<i32>>;
```

Comment:
I realize this is just moving around, but `String` is a bit weird – this isn't valid UTF-8 usually.

(Just thinking out loud, no action needed in this PR.)